lib/freedom_patches/pluck_first.rb

```diff
@@ -2,15 +2,17 @@
 
 class ActiveRecord::Relation
   def pluck_first(*attributes)
-    limit(1).pluck(*attributes).first
+    Discourse.deprecate("`#pluck_first` is deprecated, use `#pick` instead.")
+    pick(*attributes)
   end
 
   def pluck_first!(*attributes)
-    items = limit(1).pluck(*attributes)
+    Discourse.deprecate("`#pluck_first!` is deprecated without replacement.")
+    items = pick(*attributes)
 
-    raise_record_not_found_exception! if items.empty?
+    raise_record_not_found_exception! if items.blank?
```

Comment:
This behavior is different than the old one, but I am not sure it matters. Let's say the picked value is `nil` or `""`.

Previously, we had:

```
pry(main)> "".blank?
=> true
pry(main)> [""].empty?
=> false
```

I do not think it matters much especially because we will remove it in the near future.

Reply:
Just a thought... do you know used it is it in plugins?

Reply:
I see what you mean. This should probably be `items.nil?` to avoid having a change in behaviour for an empty string attribute. 👍